dashboard/src/components/AppView/ResourceTable/ResourceItem/SecretItem/SecretItemDatum.v2.tsx

```diff
@@ -33,11 +54,18 @@ function SecretItemDatum({ name, value }: ISecretItemDatumProps) {
           )}
         </button>
       </Column>
-      <Column span={11}>
-        <div className="secret-datum-text" id={`secret-item-datum-${name}-ref`}>
-          <span>
-            {name}: {hidden ? `${decodedValue.length} bytes` : <strong>{decodedValue}</strong>}
-          </span>
+      <Column span={1}>
+        <button className="secret-datum-icon" aria-expanded={!hidden} onClick={setCopiedTrue}>
+          <div data-tip={true} data-for="app-status">
+            <CopyToClipboard text={decodedValue}>
+              <CdsIcon shape="copy-to-clipboard" size="md" solid={true} />
```

Comment:
Add a `aria-label` to this icon so it notifies screen readers its purpose:

```suggestion
              <CdsIcon shape="copy-to-clipboard" size="md" solid={true} aria-label={`Copy ${name} secret value to the clipboard`}/>
```